overrides/main.html

```diff
@@ -8,4 +8,13 @@
 
 {% block libs %}
 <script src="https://cdn.usefathom.com/script.js" data-site="CUNVUWEF" data-excluded-domains="localhost,10.*.*.*,127.*.*.*,172.16.*.*,192.168.*.*" defer></script>
+{% endblock %}
+
+{% block announce %}
+<div class="announcement-bar">
+  Meet us at <a href="https://www.multipartycomputation.com/tpmpc-2022">TPMPC</a> (Aarhus, June),
```

Comment:
How about switching link color to increase visibility e.g. by adding the following to `overrides/assets/stylesheets/home.css`

```css

.announcement-bar a {
  color: var(--md-accent-fg-color)
}

.announcement-bar a:where(:focus, :hover) {
  color: var(--md-typeset-a-color);
}
```

For me the announcment loses visibility due to the contrast of the links on the background.

Reply:
Good proposal. I have implemented this as proposed.